src/techs/techs.controller.ts

```diff
@@ -477,24 +476,17 @@ export class TechsController {
         description: "forbidden - user does not have the required permission",
         type: ForbiddenErrorResponse,
     })
-    @ApiParam({
-        name: "teamId",
-        description: "voyage team Id",
-        type: "Integer",
-        required: true,
-        example: 2,
-    })
     @CheckAbilities({ action: Action.Update, subject: "TeamTechStackItem" })
-    @Patch("teams/:teamId/techs/selections")
+    @Patch("teams/techs/:techId")
     updateTechStackSelections(
         @Request() req,
-        @Param("teamId", ParseIntPipe) teamId: number,
-        @Body(ValidationPipe) updateTechSelectionsDto: UpdateTechSelectionsDto,
+        @Param("techId", ParseIntPipe) techId: number,
+        @Body(ValidationPipe) techSelectionDto: UpdateTechSelectionDto,
```

Comment:
should we leave it as updateTechSelectionDto just to be more clear that we are using the UpdateTechSelectionDto? And also consistent with the function `techs.service.ts`